docs/configuration/backends/kubernetes.md

```diff
@@ -105,14 +105,37 @@ A label selector can be defined to filter on specific Ingress objects only.
 
 See [label-selectors](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors) for details.
 
+```toml
+# Enable IngressEndpoint configuration. This will allow Traefik to update the status section of ingress objects.
+#
+# Optional
+# Default: false
```

Comment:
You could remove this line, because `kubernetes.ingressendpoint` is not a boolean.
I know that related to CLI option.